### DISCUSSION TITLE
Dark display appearance not reflected in screen

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#6] [General]
If Settings>Display & Brightness is set to Dark, then the Welcome screen that includes "Game  Overview" and "Start Game" looks odd since the buttons are dark on light, but the background is dark. 

This is not a functionality issue, but an appearance issue

**Comment by azadbolour** [comment]
Duly noted and added to the EagerWords backlog. 

I will have to research best practices for UI design with dark backgrounds and follow recommended guidelines. 

Azad
